app/services/sendgrid_mailer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class SendgridMailer
+  def self.send(to:, subsitutions:, template_id:)
+    data = {
+      personalizations: [
+        {
+          to: [
+            {
+              email: to
+            }
+          ],
+          dynamic_template_data: subsitutions
+        }
+      ],
+      from: {
+        email: "wilmanvasdez@gmail.com"
```

Comment:
Change this to a env variable something like `info@example.com` so that can be easily changed just by changing the env.